lib/pagy.rb

```diff
@@ -28,7 +28,7 @@ def initialize(vars)
       instance_variable_set(:"@#{k}", @vars.delete(k).to_i)               # set instance variables
     end
     @pages  = @last = [(@count.to_f / @items).ceil, 1].max                # cardinal and ordinal meanings
-    (1..@last).cover?(@page) || raise(OutOfRangeError, "expected :page in 1..#{@last}; got #{@page.inspect}")
+    @page >= 1 && @page <= @last or raise(OutOfRangeError, "expected :page in 1..#{@last}; got #{@page.inspect}")
```

Comment:
The Ruby style guide frowns upon the use of `or` see https://github.com/bbatsov/ruby-style-guide#no-and-or-or

Reply:
`and` and `or` are perfectly fine for control flow. I would never use them for comparing booleans, e.g. `@page >= 1 and @page <= @last` – *that’s* when bugs start to creep in.

Avdi Grimm explains it much better than I: http://www.virtuouscode.com/2014/08/26/how-to-use-rubys-english-andor-operators-without-going-nuts/